calculateDiscount.js

```diff
@@ -0,0 +1,27 @@
+let array=[]
+array=[ { name: "Milk Bottle", price: 30, discountPercent: 10},{ name: "Cheese", price: 10, discountPercent: 0},{name: "Packaged Meat (1kg)", price: 100, discountPercent: 5},{ name: "Chocolate", price: 30, discountPercent: 5},{ name: "Piza", price: 20, discountPercent: 0},{ name: "Icecream", price: 10, discountPercent: 2},{ name: "Peanut Jam", price: 10, discountPercent: 3}]
+let a=array.length;
+let finalprice=0,finalCost=0,l=0,j=1;
+
+let calculatePrice=(price,discountPercent)=>{
```

Comment:
Use `const` instead of `let` if you are not going to change it later. It is recommended to always start with using `const` instead of `let` if you do not know if it will be changed later at the time of creation of this variable.